database-postgres/data-generator/generate-publishers.js

```diff
@@ -0,0 +1,29 @@
+const faker = require('faker');
+const fs = require('fs');
+const {writeData} = require('../writeToCSV.js');
+
+// Include process module 
+const process = require('process'); 
```

Comment:
Node's `process` is a global and doesn't need to be `require`d - isn't this package a client polyfill for `process`?

Reply:
You are completely right. I had a duh moment. I used process.env without doing that, good catch!